src/deadline/max_adaptor/MaxClient/render_handlers/vray_handler.py

```diff
@@ -24,6 +26,44 @@ def __init__(self, gpu):
         """
         super().__init__()
         self.gpu: bool = gpu
+        self._validate_vray_environment()
+
+    def _validate_vray_environment(self) -> None:
+        """
+        Validates that required VRay environment variables are set.
+        Raises RuntimeError with actionable message if variables are missing.
+        """
+        # Get 3ds Max year using the same formula as max_render_submitter.py
+        max_version = rt.maxVersion()
+        # Convert to int to handle both real values and mock objects
+        version_major = int(max_version[0])
+        year = 2000 + math.ceil(version_major / 1000.0) - 2
+
+        # Define required environment variables
+        required_vars = [
+            f"VRAY_FOR_3DSMAX{year}_MAIN",
+            f"VRAY_FOR_3DSMAX{year}_PLUGINS",
+            f"VRAY_MDL_PATH_3DSMAX{year}",
+        ]
+
+        # Check for missing variables
+        missing_vars = [var for var in required_vars if var not in os.environ]
+
+        if missing_vars:
+            error_msg = (
+                f"V-Ray renderer detected, but required environment variables are missing.\n"
+                f"Please set the following variables in your host configuration script:\n"
+                f"{chr(10).join(f'  - {var}' for var in missing_vars)}"
```

Comment:
```suggestion
                f"{os.linesep.join(f'  - {var}' for var in missing_vars)}"
```

nit: this is a bit easier to read, same for below message

Reply:
sounds good.